### ISSUE TITLE
 Why no accuracy output?

### ISSUE THREAD

**Issue by tjusxh** [BVLC/caffe#1812]
In my .prototxt，there is the accuracy layer，but there is not test net and accuracy，why?
I0129 19:05:33.693763 14538 net.cpp:208] This network produces output loss
I0129 19:05:33.693788 14538 net.cpp:467] Collecting Learning Rate and Weight Decay.
I0129 19:05:33.693806 14538 net.cpp:219] Network initialization done.
I0129 19:05:33.693817 14538 net.cpp:220] Memory required for data: 1208607748
I0129 19:05:33.693892 14538 solver.cpp:41] Solver scaffolding done. 
layers {
  name: "accuracy"
  type: ACCURACY
  bottom: "fc8"
  bottom: "label"
  top: "accuracy"
  include: { phase: TEST }
}

**Comment by shelhamer** [comment]
Dupe #1810.
